djangocms_text_ckeditor/models.py

```diff
@@ -23,6 +23,13 @@ def hyphenate(t):
 @python_2_unicode_compatible
 class AbstractText(CMSPlugin):
     """Abstract Text Plugin Class"""
+
+    # Set cmsplugin_ptr and remove related_name to avoid field name clashes
+    # with any other plugin that has a field called "file".
```

Comment:
no, a field called whatever the lowercase name of the concrete subclass is called. In this case ``text``.